build.gradle

```diff
@@ -9,8 +9,8 @@ buildscript {
       'javapoet': '1.11.1',
       'kotlinpoet': '1.3.0',
       'jsr305': '3.0.2',
-      'kotlin': '1.3.40',
-      'okio': '2.3.0',
+      'kotlin': '1.3.50',
+      'okio': '2.4.0-SNAPSHOT',
```

Comment:
does this need to rely on the snapshot?

Reply:
Yes. 2.3.0 was built with 1.3.40 which includes an older, incompatible version of the Native compiler.

Reply:
But we're not going to land until there's a stable version, right?

Reply:
Yeah, we'll need another Okio release to merge this.